debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/OutboxEventRouterIT.java

```diff
@@ -121,6 +121,12 @@ public void shouldConsumeRecordsFromInsert() throws Exception {
 
         assertThat(routedEvent).isNotNull();
         assertThat(routedEvent.topic()).isEqualTo("outbox.event.user");
+
+        Struct valueStruct = requireStruct(routedEvent.value(), "test payload");
```

Comment:
That's interesting; can't see the link to the actual change here, so I'm curious about it (not to say it's bad)?

Reply:
I don't have a link... but after a bit of debugging, I realized the problem, specifically, before, the test was actually not using specific enough assertions and multiple messages would work, notice that this test asserted on anything more than the topic. This occurred  because the slot could have old state due to DROP_SLOT_ON_STOP being false. 

This change, mostly just due to adding new tests, made that different enough that it did fail some of the later tests. 

By making this test drop the slot and then have more specific assertions, we made it a lot more stable

Reply:
Ok, gotcha. Makes sense. Thanks!